.gitlab/container_build/docker_linux.yml

```diff
@@ -248,7 +275,8 @@ docker_build_fips_agent7_arm64_jmx:
   variables:
     IMAGE: registry.ddbuild.io/ci/datadog-agent/agent-base-image
     BUILD_CONTEXT: Dockerfiles/base-image
-    BUILD_ARG: --target release
+    TARGET_ARG: --target release
+    CACHE_TARGET: release
```

Comment:
Do those have an effect? The variables seem to be systematically overridden by jobs extending this base

Reply:
Hmm you do have a point here. Arguably it would be the overrides that aren't really necessary here. I feel like there's actually quite a bit of refactoring that could be done altogether. I'll merge this as is and open a refactoring PR right after.